test/integration/utils.ts

```diff
@@ -57,10 +57,11 @@ export function getTestAddresses(version: string = LATEST_ARC_VERSION): ITestAdd
   return require("@daostack/test-env-experimental/daos.json").demo[version];
 }
 
-export async function hideTrainingTooltips() {
+export async function hideTrainingTooltips(): Promise<void> {
   localStorage.setItem("trainingTooltipsEnabled", "false");
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.

Reply:
Looks like there is some problem with `webdriverio` and `typescript`, it keeps saying that `setValue` exist only on type `any`.